AndroidRunner/Device.py

```diff
@@ -123,6 +123,12 @@ def current_activity(self):
         """Returns the current focused activity on the system"""
         # https://github.com/aldonin/appium-adb/blob/7b4ed3e7e2b384333bb85f8a2952a3083873a90e/lib/adb.js#L1278
         windows = Adb.shell(self.id, 'dumpsys window windows')
+
+        recent_activity = None
+        """Newer Android 10 does not have mCurrentFocus and mFocusedApp. Different approach to get the current activity"""
+        if self.get_version() == '10':
```

Comment:
Thanks Anthony for the PR. Before accepting the PR, is it possible to unify the flow of control instead of applying your solution only for Android 10? 

As I indicated in [issue #28](https://github.com/S2-group/android-runner/issues/28), the proposed solution works also for older versions of the Android platform.

Let me know!